sql-statements/sql-statement-load-data.md

```diff
@@ -104,13 +104,16 @@ In the above example, `x'2c'` is the hexadecimal representation of the `,` chara
 
 ## MySQL compatibility
 
-* TiDB will by default commit every 20 000 rows. This behavior is similar to MySQL NDB Cluster, but not the default configuration with the InnoDB storage engine.
+This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](https://github.com/pingcap/tidb/issues/new/choose) on GitHub.
 
 > **Note:**
 >
-> Committing through splitting a transaction is at the expense of breaking the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to use `LOAD DATA` on any tables which are actively being read from or written to.
+> In earlier releases of TiDB, `LOAD DATA` committed every 20 000 rows. By default, TiDB now commits all rows in one transaction. This can result in the error `ERROR 8004 (HY000) at line 1: Transaction is too large, size: 100000058` after upgrading from TiDB 4.0 or earlier.
```

Comment:
Do you mean ``` In earlier releases of TiDB, `LOAD DATA` committed 20,000 rows of data in a transaction ```?

Reply:
It is every 20K rows. So if you are importing 101K rows, it would have been broken into 6 commits. The new behavior is always one commit. Only this is "correct" according to ACID semantics.